components/list/list-item-nav-mixin.js

```diff
@@ -66,21 +65,6 @@ export const ListItemNavMixin = superclass => class extends ListItemLinkMixin(su
 			:host([current]) .d2l-list-item-color-outer {
 				padding-block: 3px;
 			}
-
-			/* clean up with GAUD-7495-list-item-new-styles flag */
-			@supports selector(:has(a, b)) {
```

Comment:
I'm sorta guessing that the expectation was to remove this entire block.

Reply:
Correct - when initially implementing this, we had this used always for list-item-nav-mixin and then in list-item-mixin we had the same code but only used if the flag was true. Now that the flag piece of that css is cleaned up in list-item-mixin we can remove this here. [Initial PR](https://github.com/BrightspaceUI/core/pull/5782/files) for more context.